SPY/haskell-wasm
# Upgrade dependencies

Resolves the build errors due to dependencies

- Move to latest LTS resolver
- Remove upper bounds on some dependencies
- Remove `wasm.cabal` file (since the `package.yaml` autogenerates it)

The build no longer fails and the tests pass, but I don't know if the dependency bounds were there for other reasons which I may not have considered.
